File: TASK-03/elixir/subtask3.exs
IO.puts "Enter an odd number: "
{n, _} = IO.gets("") |> Integer.parse()

# Upper part
for i <- 0..div(n, 2) do
  IO.puts String.duplicate(" ", div(n, 2) - i) <> String.duplicate("*", 2 * i + 1)
end

# Lower part
for i <- (div(n, 2) - 1)..0 do
  IO.puts String.duplicate(" ", div(n, 2) - i) <> String.duplicate("*", 2 * i + 1)
end

File: TASK-03/ruby/subtask3.rb
print "Enter an odd number: "
n = gets.to_i

# Upper part
(0..n/2).each do |i|
  puts " " * (n / 2 - i) + "*" * (2 * i + 1)
end

# Lower part
(n/2 - 1).downto(0) do |i|
  puts " " * (n / 2 - i) + "*" * (2 * i + 1)
end
